git-rbr-core

```diff
@@ -152,7 +152,6 @@ upstreams as intended!
             if not ref.startswith(head_prefix):
                 non_heads.add(ref)
                 continue
-            #branch = ref[len(head_prefix):]
```

Comment:
Sure!

Note the comment at the top of the function :) :
>      # TODO this function's code is a bit messy.

This is a nice small step in fixing that.